client/post-editor/media-modal/gallery/edit.jsx

```diff
@@ -40,24 +41,44 @@ export default React.createClass( {
 	},
 
 	render() {
-		const { site, settings } = this.props;
+		const { onUpdateSetting, site, settings, translate } = this.props;
 
 		if ( ! site || ! settings.items ) {
 			return null;
 		}
 
+		const orders = {
+			[ translate( 'Reverse order' ) ]: reverse( [ ...settings.items ] ),
+			[ translate( 'Order alphabetically' ) ]: sortBy( settings.items, 'title' ),
+			[ translate( 'Order chronologically' ) ]: sortBy( settings.items, 'date' ),
```

Comment:
Good question. In the strict sense of the word I'd say yes -- newest first would be _inverse chronologically_ ordered, no? For some use cases I can think of, actual chronological order does make sense -- e.g. when showcasing photos from an event, it makes sense to start at the beginning.

So I think I'll stick with it -- easy enough to press 'Reverse order' afterwards if it's not the desired result.